web-app/src/app/(landing)/components/navigation-menu.tsx

```diff
@@ -13,22 +13,22 @@ const navigationLinks: NavigationLinkData[] = [
   {
     href: "/agents",
     labelKey: "agents",
-    showChevron: true,
```

Comment:
I believe we don't need the `showChevron` anymore. The entire logic can be removed.

Reply:
a3cfcb6f2ef48ddbbb3da3518916b48adae75349